spec/models/responsible_body_spec.rb

```diff
@@ -450,6 +450,159 @@
         expect(school_c.raw_laptops_full).to eq([20, 0, 0, 0])
       end
     end
+
+    context 'case 4' do
+      subject(:rb) { create(:trust, :manages_centrally, :vcap) }
+
+      let!(:school_a) { create(:school, :in_lockdown, :centrally_managed, responsible_body: rb, laptops: [10, 0, 0, 0]) }
+      let!(:school_b) { create(:school, :in_lockdown, :centrally_managed, responsible_body: rb, laptops: [10, 0, 0, 0]) }
+      let!(:school_c) { create(:school, :centrally_managed, responsible_body: rb, laptops: [10, 0, 20, 30]) }
+
+      it 'do not touch over-ordered distribution when there is no room for redistribution' do
```

Comment:
It "does not touch over-ordered distribution when there is no capacity for redistribution"